libc/src/setjmp/setjmp_impl.h

```diff
@@ -13,9 +13,13 @@
 // public header setjmp.h which is also included. here.
 #include "hdr/types/jmp_buf.h"
 #include "src/__support/macros/config.h"
+#include "src/__support/macros/properties/compiler.h"
 
 namespace LIBC_NAMESPACE_DECL {
 
+#ifdef LIBC_COMPILER_IS_GCC
+[[gnu::nothrow]]
```

Comment:
Nit: do you mind adding comments here about gcc implicitly add `nothrow` with `[[gnu::naked]]`?

Reply:
done in https://github.com/llvm/llvm-project/pull/112415/commits/cef25394b9e09e84f3a0c484f5fc343d61581063